### ISSUE TITLE
Client to server should not be required for server to server

### ISSUE THREAD

**Issue by cwebber** [w3c-social/activitypump#184]
This is legacy from the verrrry beginnings of the document, back when it was oshepherd's work:

> 2.1 Specification Profiles
> 
> This specification defines two closely related and interacting protocols:
>
> * A client to server protocol, or "Social API"
>   This protocol permits a client to act on behalf of a user. For example, this protocol is used by a mobile phone application to interact with the user's social stream. 
> * A server to server protocol, or "Federation Protocol"
>   This protocol is used to distribute activities between users on different servers, tying them into the same social graph. 
>
>All servers claiming conformance to this specification are required to implement the former protocol, and may optionally implement the latter.

We've worked hard to separate the client to server and server to server sections into being entirely optional and separate.  It shouldn't be mandatory to implement client to server to implement server to server.

**Comment by cwebber** [comment]
Proposed change:

``` diff
         <p>
-          All servers claiming conformance to this specification are required to
-          implement the former protocol, and may optionally implement the latter.
+          The ActivityPub specification is designed so that implementing either
+          of these protocols means supporting the other is of very little
+          additional effort, but servers may implement one without the other.
           This gives three conformance classes:
         </p>
```

(I'm open to wording adjustments if people have them.)

**Comment by evanp** [comment]
That's the right way to do it.

**Comment by strugee** [comment]
Bit of a run-on and a little hard to parse (at least for me), how about:

> The ActivityPub specification is designed so that once either of these protocols are implemented, supporting the other is of very little additional effort. However, servers may still implement one without the other.

**Comment by sandhawke** [comment]
I support the theory that this isn't a substantive change because (a) it doesn't seem to invalidate any of the comments we've received, (b) it's unlikely this change would make any supporter stop supporting it, and (c) it wouldn't actually make anyone change their code.     It is something of a judgment call, though.

**Comment by cwebber** [comment]
It also wouldn't break anything that is already compliant (thought it may make something compliant that currently isn't).

**Comment by tantek** [comment]
Another way to look at it is, yes normative change but has no impact on any existing implementations.

**Comment by dissolve** [comment]
It doesn't even change conformance classes for any existing conforming implementation, It just relaxes one of the conformance classes to allow implementations that were not conformant previously.

**Comment by rhiaro** [comment]
Also

``` diff
- This designation applies to any implementation of the entirety of the server 
- portion of both the client to server and federation protocols.
+ This designation applies to any implementation of the entirety of the federation 
+ protocols. 
```
